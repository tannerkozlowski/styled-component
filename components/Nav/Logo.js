import styled from 'styled-components'

import rem from '../../utils/rem'

const Logo = styled.div`
  display: inline-block;
  vertical-align: middle;
  box-sizing: border-box;
  width: ${rem(164)};
  height: ${rem(40)};

  background-image: url(/static/nav-logo.png);
  background-position: center;
  background-size: contain;
`

export default Logo
