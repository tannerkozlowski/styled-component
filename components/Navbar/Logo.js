import styled from 'styled-components'
import rem from 'polished/lib/helpers/rem'

const Logo = styled.div`
  display: inline-block;
  width: 100%;
  height: 100%;

  box-sizing: border-box;

  background-image: url(/static/logo.png);
  background-size: contain;
  background-position: center;

  @media (max-width: 1000px) {
    background-image: url(/static/icon.png);
    width: ${rem(70)};
  }
`

export default Logo
