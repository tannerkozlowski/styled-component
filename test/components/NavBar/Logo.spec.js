import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Logo from '../../../components/Nav/Logo'

test('Logo renders correctly', () => {
  const tree = renderer.create(<Logo />).toJSON()

  expect(tree).toMatchSnapshot()
})
