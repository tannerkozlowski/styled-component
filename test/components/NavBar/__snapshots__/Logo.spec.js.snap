// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
.c0 {
  display: inline-block;
  box-sizing: border-box;
  width: 9.11111111111111rem;
  height: 2.2222222222222223rem;
  background-image: url(/static/nav-logo.png);
  background-position: center;
  background-size: contain;
}

<div
  className="c0"
/>
`;
